README.md

```diff
@@ -3,7 +3,7 @@
 <!-- Essentials -->
 
 [![PyPI](https://img.shields.io/pypi/v/glass)](https://pypi.org/project/glass)
-[![Documentation](https://readthedocs.org/projects/glass/badge/?version=latest)](https://glass.readthedocs.io/latest)
+[![Documentation](https://readthedocs.org/projects/glass/badge/?version=stable)](https://glass.readthedocs.io/stable)
```

Comment:
In the README, which everyone will generally see on current main, this should point to `latest` not `stable` IMO.

Reply:
Hmm... I guess we want the badge to refer to `stable`. But are you sure we want the docs link to be `latest`? People will in general be using the most recent stable version, so the docs could become out of sync.

I thought it would be off to have `latest` for the badge, and `stable` for the link - which is why I had them both as `stable`.

Reply:
Yeah I guess the question here is who will be looking at the repo. Probably even people who are on stable, so maybe you are right that stable is better.

Reply:
I left `CONTRIBUTING.md` as `/latest` because it will be developers looking at that